bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -87,6 +88,27 @@ <h2 class="mzp-has-zap-7">{{ ftl('firefox-desktop-download-get-the-browser') }}<
         </div>
       </div>
       <div class="c-block-media l-v-center">
+      {% if switch('firefox-mr1-launch') %}
+        <div class="c-block-img">
```

Comment:
The opening div with class `c-block-img` should be outside the if/else. Or its closing tag should be inside the if/else block. Right now there's two opening tags on one closing tag and it makes me nervous 😬